tox.ini

```diff
@@ -15,7 +15,6 @@ setenv =
   PYTHONPATH = {toxinidir}:{toxinidir}/src/:{toxinidir}/reactive/:{toxinidir}/hooks/:{toxinidir}/lib/:{toxinidir}/actions:{toxinidir}/files/:{toxinidir}/files/plugins/
   # avoid state written to file during tests - see https://github.com/juju/charm-helpers/blob/85dcbeaf63b0d0f38e8cb17825985460dc2cd02d/charmhelpers/core/unitdata.py#L179-L184
   UNIT_STATE_DB = :memory:
-  TEST_JUJU3 = 1
```

Comment:
what this project does not pick up the change from automation repo?

Reply:
Ah, maybe a race condition: this commit was created before https://github.com/canonical/charmed-openstack-upgrader/pull/617 is merged